baus/subsidies.py

```diff
@@ -398,6 +398,10 @@ def policy_modifications_of_profit(feasibility, parcels):
                         orca.get_injectable("scenario") in \
                         policy["alternate_geography_scenarios"]:
                     formula = policy["alternate_adjustment_formula"]
+                elif "geography_scenarios_fb" in policy and \
+                        orca.get_injectable("scenario") in \
+                        policy["geography_scenarios_fb"]:
+                    formula = policy["profitability_adjustment_formula_fb"]
                 else:
                     formula = policy["profitability_adjustment_formula"]
```

Comment:
@yuqiww so this setup means that s23 and s24 will activate their reduction policies, but if the scenarios are removed from `alternate_geography_scenarios` or `profitability_adjustment_formula_fb` they will default to the `profitability_adjustment_formula`. 
Would `profitability_adjustment_formula` work in this case, and is it a "baseline" or anything like that? If not, maybe we just don't use it. Any other scenario in `enable in scenarios` would also try to use it. So we'd want to change the code if other scenarios need a default.

Reply:
I'm remembering that `profitability_adjustment_policy` was probably set up when we had s21 and s22. So new scenarios will just need their own settings, like you did here. Okay I'm caught up!